libvmi/json_profiles/json_profiles.c

```diff
@@ -100,6 +105,27 @@ status_t vmi_get_symbol_addr_from_json(vmi_instance_t vmi, json_object* json, co
     return vmi->json.handler(json, symbol, NULL, addr, NULL);
 }
 
+status_t vmi_address_to_symbol_from_json(vmi_instance_t vmi, addr_t addr, const char** symbol)
+{
+    if(vmi->json.build_reverse_symbol_table == NULL)
+    {
+        dbprint(VMI_DEBUG_MISC, "Address to symbol not implemented for this type of json profile\n");
+        return VMI_FAILURE;
+    }
+
+    /* delay table build until feature is used */
```

Comment:
Instead of this being a delayed step this should be performed by adding a new `VMI_INIT` flag and be built during init time.

Reply:
Removed delayed ht build and added `VMI_INIT_SYMBOLLOOKUP `

Reply:
I did not find a name for  `VMI_INIT_...` that's more expressive but still concise. Alternatively, what do you think about performing the ht build by default? The overhead is not too high.

These are execution times for a win10 volatility profile:
```
build_reverse_symbol_table() took 20.645000 ms to execute 
vmi_init_profile() took 215.098000 ms to execute 
```
In the measurement `build_reverse_symbol_table() ` was executed a part of` vmi_init_profile()`

Reply:
I wouldn't want it on by default so either an initflag or a compile option would be required. I think the init flag is better. Let's go with `VMI_INIT_V2SYM`, that's more consistent with the API naming.

Reply:
Changed to `VMI_INIT_V2SYM`